Tests/InstrumentationTests/InstrumentationSystemTests.swift

```diff
@@ -12,26 +12,43 @@
 //===----------------------------------------------------------------------===//
 
 import Baggage
-import Instrumentation
+@testable import Instrumentation
 import XCTest
 
 final class InstrumentationSystemTests: XCTestCase {
     func testItProvidesAccessToASingletonInstrument() {
         let tracer = FakeTracer()
+        let instrument = FakeInstrument()
+        let multiplexInstrument = MultiplexInstrument([tracer, instrument])
 
-        InstrumentationSystem.bootstrap(tracer) // FIXME, must use bootstrapInternal
-        XCTAssertTrue(InstrumentationSystem.instrument as? FakeTracer === tracer)
+        XCTAssertNil(InstrumentationSystem.instance(of: FakeTracer.self))
+        XCTAssertNil(InstrumentationSystem.instance(of: FakeInstrument.self))
+
+        InstrumentationSystem.bootstrapInternal(multiplexInstrument)
+        XCTAssert(InstrumentationSystem.instrument is MultiplexInstrument)
+        XCTAssert(InstrumentationSystem.instance(of: FakeTracer.self) === tracer)
+        XCTAssert(InstrumentationSystem.instance(of: FakeInstrument.self) === instrument)
+
+        InstrumentationSystem.bootstrapInternal(tracer)
+        XCTAssertFalse(InstrumentationSystem.instrument is MultiplexInstrument)
+        XCTAssert(InstrumentationSystem.instance(of: FakeTracer.self) === tracer)
+        XCTAssertNil(InstrumentationSystem.instance(of: FakeInstrument.self))
     }
```

Comment:
i see that's pretty good actually

Reply:
how about calling it `instrument(of:` tbh? And we'll need also `.instrument` and `tracingInstrument` as convenience 

 so `instrument` gets the whatever was set



tracing instrument is an implicit first where is `TracingInstrument`

Reply:
+1 `.tracer` is already implemented this way in `TracingInstrument`, but I'll rename it to `tracingInstrument`.